packages/client/src/hooks/useModal.tsx

```diff
@@ -0,0 +1,14 @@
+import { useState } from 'react'
+
+export const useModal = () => {
+  const [isOpen, setIsOpen] = useState(false)
+
+  const toggle = () => {
+    setIsOpen(!isOpen)
```

Comment:
[Исправление]
Когда новое значение стейта зависит от предыдущего, лучше записывать вот так:
```suggestion
    setIsOpen(prev => !prev)
```
Есть [вот такое на стековерфлоу](https://stackoverflow.com/questions/55495198/reacts-setstate-method-with-prevstate-argument)